testdata/cheats/SignP256.t.sol

```diff
@@ -0,0 +1,18 @@
+// SPDX-License-Identifier: MIT OR Apache-2.0
+pragma solidity 0.8.18;
+
+import "ds-test/test.sol";
+import "./Vm.sol";
+
+contract SignTest is DSTest {
+    Vm constant vm = Vm(HEVM_ADDRESS);
+
+    function testSignP256() public {
+        bytes32 pk = hex"0A1D0F90D7299AF6F990FD67A2D535D2FD82BCDA9B9AA352E08B262C7021573C";
+        bytes32 digest= hex"54705ba3baafdbdfba8c5f9a70f7a89bee98d906b53e31074da7baecdc0da9ad";
+
+        (bytes32 r, bytes32 s) = vm.signP256(uint256(pk), digest);
```

Comment:
Getting `[FAIL. Reason: unknown selector `0x83211b40` for VmCalls]` not sure what's wrong

Reply:
It should work, CI fails after in the `assertEq`s.

Reply:
ah, wasn't running cargo bin forge. Also test was broken because code was broken :)